templates/hosts.j2

```diff
@@ -1,21 +1,6 @@
-#
-# /etc/hosts: static lookup table for host names
-#
-
 # {{ ansible_managed }}
 
-# localhost
-127.0.0.1 localhost localhost.{{ common_domain }}
-::1 localhost localhost.{{ common_domain }}
-
-# IPv4
-{% for addr in ansible_all_ipv4_addresses %}
-	{{- addr }} {{ inventory_hostname_short }}.{{ common_domain }} {{ inventory_hostname_short }}
-{% endfor %}
-
-# IPv6
-{% for addr in ansible_all_ipv6_addresses %}
-	{{- addr }} {{ inventory_hostname_short }}.{{ common_domain }} {{ inventory_hostname_short }}
-{% endfor %}
-# End of file
-
+127.0.0.1 localhost
+::1 localhost
+127.0.0.1 {{ inventory_hostname_short }}
+::1 {{ inventory_hostname_short }}
```

Comment:
In edge cases having the same IP address in multiple lines might cause problems. [1]

```suggestion
127.0.0.1 localhost  {{ inventory_hostname_short }}
::1 localhost {{ inventory_hostname_short }}
```

[[1]](https://unix.stackexchange.com/questions/102660/hosts-file-is-it-incorrect-to-have-the-same-ip-address-on-multiple-lines)